quickmenu/arm9/source/main.cpp

```diff
@@ -970,7 +970,7 @@ int main(int argc, char **argv) {
 
 		snprintf(pictochatPath, sizeof(pictochatPath), "/_nds/pictochat.nds");
 		pictochatFound = (access(pictochatPath, F_OK) == 0);
-		if (!pictochatFound) {
+		if (!pictochatFound && consoleModel == 0) {

```

Comment:
I find it easier to read if constants are used here for the different console models.

Reply:
> I find it easier to read if constants are used here for the different console models.

Ha, so how to do. I have no idea currently:P